kopi/assetlist.json

```diff
@@ -163,13 +163,13 @@
       "base": "ucusdtinj",
       "name": "cUSDT.inj",
       "display": "cUSDT.inj",
-      "symbol": "CUSDT.inj",
+      "symbol": "`CUSD  `T.inj",
```

Comment:
we can't have this value for symbol. Please use either all UPPERCASE or can have some lowercase and certain symbols if traces is defined.